etna/core/models/settings.py

```diff
@@ -12,7 +13,35 @@
 
 @register_setting(icon="list-ul")
 class SiteSettings(BaseSetting, ClusterableModel):
+    beta_banner_standfirst = models.CharField(
+        max_length=200,
+        default="What does the 'beta' label mean?",
+        verbose_name="standfirst",
+    )
+    beta_banner_link = models.URLField(blank=True, verbose_name="link to")
+    beta_banner_link_text = models.CharField(
+        max_length=200,
+        default="Find out more ?",
```

Comment:
where did this question mark come from, @mozaki1? probably best that we lose it:
```suggestion
        default="Find out more",
```